### PR TITLE
Db/component content size

### PR DESCRIPTION
**Pull request by goransle**

### CODE REVIEW

File: ts/Dashboard/Component/Component.ts [hunk 1]
```diff
@@ -584,8 +584,12 @@ abstract class Component<TEventObject extends Component.EventTypes = Component.E
...
         if (!this.hasLoaded) {
             this.load();
-            // Call resize to fit to the cell
-            this.resizeTo(this.parentElement);
+            // Call resize to fit to the cell. Timeout was added to make sure
+            // that each element is added to the DOM and that we have the
+            // correct dimensions.
+            setTimeout((): void => {
+                this.resizeTo(this.parentElement);
+            }, 100);
```
Comment (bre1470) [inline ts/Dashboard/Component/Component.ts:592]: If you are waiting for the DOM refresh cycle, a timeout of `0` is enough. If you need more than that, the timeout concept is not the best solution and `MutationObserver` should be used instead.

**Review by bre1470 — CHANGES_REQUESTED** [review]

File: ts/Dashboard/Component/Component.ts (hunk 1 above)
Comment (karolkolodziej) [reply ts/Dashboard/Component/Component.ts]: Solved easier than expected.

**Review by bre1470 — APPROVED** [review]
Looks good! 👍

**Review by pawellysy — APPROVED** [review]
LGTM 👍

**Pull request merged**